server/src/main/java/io/deephaven/server/appmode/ApplicationServiceGrpcImpl.java

```diff
@@ -304,7 +378,7 @@ public void run() {
         // must be sync wrt parent
         private void markRunning() {
             if (!isScheduled) {
-                throw new IllegalStateException("Job is running without being scheduled");
+                // throw new IllegalStateException("Job is running without being scheduled");
```

Comment:
Why did you need to comment this?

Reply:
It didn't work w/ calling propogateUpdates on thread. I could make the state management better for on-thread vs scheduler tracking, but I think it's a potentially unnecessary safety belt.